test/test_common/test_runtime.h

```diff
@@ -0,0 +1,49 @@
+// A simple test utility to easily allow for runtime feature overloads in unit tests.
+//
+// As long as this class is in scope one can do runtime feature overrides:
+//
+//  TestScopedRuntime scoped_runtime;
+//  Runtime::LoaderSingleton::getExisting()->mergeValues(
+//      {{"envoy.reloadable_features.test_feature_true", "false"}});
+
+#pragma once
+
+#include "common/runtime/runtime_impl.h"
+#include "common/stats/isolated_store_impl.h"
+
+#include "test/mocks/event/mocks.h"
+#include "test/mocks/init/mocks.h"
+#include "test/mocks/local_info/mocks.h"
+#include "test/mocks/protobuf/mocks.h"
+#include "test/mocks/runtime/mocks.h"
+#include "test/mocks/thread_local/mocks.h"
+
+#include "gmock/gmock.h"
+
+namespace Envoy {
+
+// TODO(alyssawilk) move existing runtime tests over to using this.
+class TestScopedRuntime {
+public:
+  TestScopedRuntime() : api_(Api::createApiForTest()) {
+    envoy::config::bootstrap::v2::LayeredRuntime config;
+    config.add_layers()->mutable_admin_layer();
```

Comment:
Can you add a small comment that this is required for mergeValues() to work? This wasn't immediately clear to me.

Reply:
Sorry I meant to add a comment above this line along the lines of "the existence of an admin layer is required for mergeValues() to work"